.golangci.yml

```diff
@@ -1,204 +1,173 @@
-  gosec:
-    includes:
-    - G101
-    - G102
-    - G103
-    - G104
-    - G106
-    - G108
-    - G109
-    - G111
-    - G113
-    - G201
-    - G202
-    - G203
-    - G303
-    - G307
-    - G403
-    - G502
-    - G503
-    - G504
-    - G601
-  revive:
-    rules:
-    - name: package-comments
-      disabled: true
-    - name: error-strings
-      disabled: true
-    - name: unexported-return
-      disabled: true
-  govet:
-    enable-all: true
-    disable:
-      - shadow
-      - fieldalignment
-    settings:
-      printf: # analyzer name, run `go tool vet help` to see all analyzers
-        funcs: # run `go tool vet help printf` to see available settings for `printf` analyzer
-          - Print
-          - Printf
-          - Println
-          - Debug
-          - Debugf
-          - Info
-          - Infof
-          - Warn
-          - Warnf
-          - Error
-          - Errorf
-          - github.com/stackrox/rox/migrator/log.WritetoStderr
-          - github.com/stackrox/rox/migrator/log.WritetoStderrf
-  gocritic:
-    disabled-checks:
-    - appendAssign
-    - argOrder
-    - assignOp
-    - captLocal
-    - dupArg
-    - elseif
-    - exitAfterDefer
-    - ifElseChain
-    - mapKey
-    - singleCaseSwitch
-    - unlambda
-    - wrapperFunc
-  nolintlint:
-    allow-leading-space: false # require machine-readable nolint directives (i.e. with no leading space)
-    allow-unused: false # report any unused nolint directives
-    require-explanation: false # don't require an explanation for nolint directives
-    require-specific: true # require nolint directives to be specific about which linter is being skipped
-  forbidigo:
-    # Forbid the following identifiers (list of regexp).
-    forbid:
-    - ^print\(.*\)$
-    - 'fmt\.Print.*(# Disallowed function used\. Use environments functions for printing or to a specific writer from environment\.InputOutput\(\)\.)?'
-    - 'os\.Stdout(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?'
-    - 'os\.Stderr(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?'
-    - 'os\.Stdin(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?'
-  staticcheck:
-    checks: [all,-ST1000,-ST1001,-ST1003,-ST1005,-SA1019,-SA4001,-ST1016]
-  wrapcheck:
-    ignoreSigRegexps:
-      - utils\.Should
-      - errox\..+\.CausedBy(f)?
-      - retry\.MakeRetryable
-      - policy\.NewErr.*
-
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
-  disable-all: true
+  default: none
   enable:
     - asciicheck
-    # - bodyclose
     - copyloopvar
-    # - deadcode
-    # - depguard
-    # - dogsled
-    # - dupl
     - errcheck
-    # - funlen
     - forbidigo
-    # - gochecknoglobals
-    # - gochecknoinits
-    # - gocognit
-    # - goconst
```

Comment:
Applies to the whole `linters/enable` section: Keep the commented out linters from the list (or at least the ones with comments hinting at follow-up actions.
As a follow-up, we could check the current list of linters and check if some would be of interest.